### ISSUE TITLE
pausing abuse , and low behaviour players cant unpause

### ISSUE THREAD

**Issue by perathion** [ValveSoftware/Dota2-Gameplay#11151]
### Description

players above 3k behaviour score can freely pause the game for an infinite amount of time while low score players can not even unpause , i do undrstand why low score players can not pause , but why unpausing is banned? this is really fked up volve , i am stuck in a game and cant do anything about it 
fix it asap

**Comment by Shiftt1** [comment]
YES FIX IT ASAP WE ARE SITTING FOR 3 HOURS ALREADY AND IT's SECOND TIME HAPPENED
